README/ReleaseNotes/v628/index.md

```diff
@@ -182,12 +182,42 @@ RNTuple is still experimental and is scheduled to become production grade in 202
 ### New features
 
 - Add [`GraphAsymmErrors`](https://root.cern/doc/master/classROOT_1_1RDF_1_1RInterface.html#acea30792eef607489d498bf6547a00a6) action that fills a TGraphAsymmErrors object.
+- Introduce [`RDatasetSpec`](https://root.cern/doc/master/classROOT_1_1RDF_1_1Experimental_1_1RDatasetSpec.html) as an
```

Comment:
sadly there are no docs at this link or at https://root.cern/doc/master/classROOT_1_1RDataFrame.html that explain how to use RDatasetSpec, we'll have to fix that